cli/secret/secret_add.go

```diff
@@ -46,17 +46,13 @@ var secretCreateCmd = &cli.Command{
 			Usage: "secret value",
 		},
 		&cli.StringSliceFlag{
-			Name:  "event",
```

Comment:
could we use alias to still have old one too?

Reply:
I marked this PR as breaking and added a migration note to the docs page.